component/src/main/java/com/celements/blog/article/Article.java

```diff
@@ -445,4 +486,183 @@ public String toString() {
     return "Article [docRef=" + getDocumentReference() + "]";
   }
 
+  public String getExtractPlainTextEncoded(String lang, boolean isViewtypeFull, int maxNumChars) {
+    String articleExtract = getExtract(lang, isViewtypeFull, maxNumChars);
+    try {
+      String plainExtract = new PlainTextCommand().convertHtmlToPlainText(articleExtract);
+      return plainExtract.replaceAll("\"", "&quot;");
+    } catch (ConvertToPlainTextException ctpte) {
+      LOGGER.error("Failed to convert article extract to plain text for article {}", articleDocRef,
+          ctpte);
+    }
+    return articleExtract;
+  }
+
+  public List<ImageUrlDim> getArticleImagesBySizeAsc(String lang) {
```

Comment:
this is a public method with a non-public nested class in the signature, don't do this! either remove modifier (if only used internally) or change ImageUrlDim to public